src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class TaskListTest {
+
+    @Test
+    public void add_task_taskAdded() {

```

Comment:
Even though this is a test case but should these methods be named as camel cases instead? I noticed the same issue with methods naming in other test cases as well.
```suggestion
    public void addTaskTaskAdded() {

```

Reply:
Thanks for highlighting!